routes/property.js

```diff
@@ -53,6 +60,26 @@ const getFilteredProperty = async (req, res, next) => {
     }
 } 
 
+const getIsWishlisted = (token, propertyId) => {
+    return new Promise((resolve, reject) => {
+        if(!token) resolve(false);
+        else {
+            jwt.verify(token, process.env.SECRET_KEY, (err, decoded) => {
+                if(err) {
+                    reject(err);
+                }
+                const { userId } = decoded;
+                getWishlistForUser(propertyId, userId).then(result => {
+                    if(result.length == 0) resolve(false);
+                    else resolve(true);
+                }).catch(e => {
+                    reject(e);
+                })
+            });
+        }
+    })
+}
+
```

Comment:
we could use something like this https://github.com/pranavpandey1998official/serverSEN/blob/c13fdba6fa8a2ace0e6a91ece1c015ab5796357b/routes/property_review.js#L115 and avoid all this boiler plate

Reply:
but here if the token is not present then it will directly send back telling its not authorised and it wont proceed forward with the request

Reply:
cool, but there are still merge conflicts if you cannot resolve them just delete the whole file. we don't need this

Reply:
so should i delete package-lock file and then make a pull request

Reply:
yes